### PR TITLE
Show bench percentage

### PR DESCRIPTION
**Pull request by npraderi**

### CODE REVIEW

**Review by SBentancourt — CHANGES_REQUESTED** [review]

File: src/state/recoil/selectors/benchPercentageSelector.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    );
+
+    const benchHours = benchMembers.reduce(
+      (hoursResult, benchMember) => hoursResult + benchMember.hours,
+      0
+    );
+
+    const totalPercentage = Math.round((benchHours / allMembersHours) * 100);
+
+    return totalPercentage;
```
[the remark this answers is not in the document]
Comment (npraderi) [reply src/state/recoil/selectors/benchPercentageSelector.ts:30]: Sure !

**Review by SBentancourt — DISMISSED** [review]

File: src/utils/dnd.ts [hunk 2]
```diff
@@ -28,7 +30,7 @@ export const getPositionAfterDrop = ({
   const position = applyReverseScale(
     {
       x: initialPosition.x + delta.x,
-      y: initialPosition.y + delta.y,
+      y: initialPosition.y - (TOOLBAR_HEIGHT + NAVBAR_HEIGHT) + delta.y,
```
Comment (juanlarrambebere) [inline src/utils/dnd.ts:33]: Don't we need to the the same on the drag preview layer? I think we are not using this utility method for drag previews cause the math is a little bit different.

**Review by juanlarrambebere — COMMENTED** [review]
Looks good though I left you a question regarding drag previews

File: src/utils/dnd.ts (hunk 2 above)
Comment (juanlarrambebere) [reply src/utils/dnd.ts:33]: Hmm I tested it and the drag preview works fine, so never mind I guess!

**Review by juanlarrambebere — APPROVED** [review]

File: src/utils/dnd.ts (hunk 2 above)
Comment (npraderi) [reply src/utils/dnd.ts:33]: Actually, I did the same on CustomDragLayer if that is what you mean

**Pull request merged**